### DISCUSSION TITLE
Look at Gnome 40

### DISCUSSION THREAD

**Discussion by jangernert** [elementary/streams#32] [Ideas]
- Install & test the latest builds of gnome 40
- General thoughts
- Compare how certain problems were solved compared to elemantary
  - Things elementary & gnome 40 are doing different and reasons for both approaches
  -  (educated guess in case of gnome 40)
- Were parts of gnome 40 inspired by elementary
- Does gnome 40 bring some new ideas that elementary wants to explore as well?

**Comment by cassidyjames** [comment]
Huh, that's really interesting actually, thanks for filing! This is something I kind of do privately anyway, so it could be interesting to capture it on a stream.

**Comment by Siilwyn** [comment]
Yes I'm on board. ⛵

**Comment by alatiera** [comment]
You could use the nightly build from https://os.gnome.org/ for this, almost everything is merged and we push updates to the ostree repo twice a day. If something doesn't work in the iso feel free to poke me!
